libcontainer/cgroups/fs/cpuset.go

```diff
@@ -48,7 +50,109 @@ func (s *CpusetGroup) Remove(d *cgroupData) error {
 	return removePath(d.path("cpuset"))
 }
 
+func getNumericStat(path string, filename string, extracted *[]uint16) error {
```

Comment:
The name is way too generic. Perhaps `getCpusetStat` or something like that?

Reply:
Applied